### ISSUE TITLE
Breaks with vite-ssg on vercel build

### ISSUE THREAD

**Issue by appinteractive** [aleclarson/vite-tsconfig-paths#31]
Running build localy works great, but when pushing to vercel it breaks.

```
00:18:34.586  	vite v2.4.1 building SSR bundle for production... 
00:18:34.633  	 > node_modules/vite-tsconfig-paths/dist/index.js:103:40: error: [plugin: vite:dep-scan] viteResolve is not a function
```

**Comment by lukasluecke** [comment]
I have the same issue - not sure how to reproduce it locally either :/

Have switched to manual alias definitions for now, but would prefer to use this plugin.

**Comment by cuibonobo** [comment]
Also seeing this same error on Cloudflare Pages on Node 14.

**Comment by jamesknelson** [comment]
Having this issue too. It turns out it's possible to reproduce it locally by deleting the `node_modules/.vite` directory.

Running `vite dev` to recreate the directory fixes the issue, and then removing it again makes the issue return.

**Comment by aleclarson** [comment]
Fixed in v3.3.14
